trin-history/src/jsonrpc.rs

```diff
@@ -170,6 +170,17 @@ impl HistoryRequestHandler {
 
                     let _ = request.resp.send(response);
                 }
+                HistoryEndpoint::RecursiveFindNodes(node_id) => {
+                    let node_id = discv5::enr::NodeId::from(node_id.0);
+                    let mut nodes = self.network.overlay.lookup_node(node_id).await;
+                    nodes.sort_by(|a, b| {
+                        XorMetric::distance(&node_id.raw(), &a.node_id().raw())
+                            .cmp(&XorMetric::distance(&node_id.raw(), &b.node_id().raw()))
+                    });
+                    let nodes: Vec<Enr> = nodes.into_iter().take(16).collect();
```

Comment:
16 is used across a few methods that have this behavior. It probably belongs in some rpc constants file. I don't see one yet, and this is the only method written so far (there's going to be the same methods for state and basic discv5). I am fine punting for now I suppose.

Reply:
Yeah, seems like it's fine to save this consolidation until we build out the next network and have a more clear idea of where to house things like these constants.